autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -260,6 +260,11 @@ def prepare(
         if self.use_snapshot_ensemble:
             self.model_snapshots = list()
 
+        # in case we are using, swa or se with early stopping,
+        # we need to make sure network params are only updated
+        # from the swa model if the swa model was actually updated
+        self.swa_updated: bool = False
+
```

Comment:
It seems no description about what SWA (stochastic weighed average?) means, so could you add the description in the documentation of this class?

Reply:
Sure